frontend/src/screens/wallet/Receive.tsx

```diff
@@ -16,6 +16,7 @@ export default function Receive() {
   const { toast } = useToast();
   const navigate = useNavigate();
 
+  // TODO: Include offer if not logged in
```

Comment:
This should be done, right? (does https://github.com/getAlby/hub/pull/1277 need to be merged first?)